hypertrace-core-graphql-rx-utils/src/main/java/org/hypertrace/core/graphql/rx/NetworkScheduler.java

```diff
@@ -0,0 +1,15 @@
+package org.hypertrace.core.graphql.rx;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import javax.inject.Qualifier;
+
+@Qualifier
+@Target({FIELD, PARAMETER, METHOD})
+@Retention(RUNTIME)
+public @interface NetworkScheduler {}
```

Comment:
Who is using this annotation?

Reply:
This is used to disambiguate when injecting a scheduler, since we generally want to pick a specific implementation - i.e.:
```java
@Inject
  MyClass(@NetworkScheduler Scheduler networkScheduler) {
```
Here's a full example: https://github.com/google/guice/wiki/BindingAnnotations#defining-binding-annotations . 

An alternative is using something like `@Named("NetworkScheduler") Scheduler networkScheduler`, but since we have to pass a constant anyway it doesn't seem to make much of a different using a string vs an annotation.